src/syscheckd/db/src/fimDB.cpp

```diff
@@ -0,0 +1,252 @@
+/*
+ * Wazuh SysCollector
+ * Copyright (C) 2015-2021, Wazuh Inc.
+ * September 27, 2020.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#include "fimDB.hpp"
+
+#define FIM_LOCATION      "syscheck"
```

Comment:
Why this definition is located here? I see this as too generic and could be needed in other source files. Could we add it to the commonDefs file?

Reply:
The commonDefs is being worked by Antonio in a different PR, he will be in charge of collecting all the common definitions.